src/payment/strategies/paypal-commerce/paypal-commerce-payment-strategy.ts

```diff
@@ -120,4 +122,29 @@ export default class PaypalCommercePaymentStrategy implements PaymentStrategy {
             intent,
         };
     }
+
+    private _getFindingKey(methodId: string): keyof PaypalCommerceSDKFunding {
+        if (this._typePayment === PaymentStrategyTypes.credit) {
+            return 'PAYLATER';
+        } else if (this._typePayment === PaymentStrategyTypes.alternative) {
+            switch (methodId) {
+                case 'bancontact':
+                    return 'BANCONTACT';
+                case 'giropay':
+                    return 'GIROPAY';
+                case 'przelewy24':
+                    return 'P24';
+                case 'eps':
+                    return 'EPS';
+                case 'ideal':
+                    return 'IDEAL';
+                case 'mybank':
+                    return 'MYBANK';
+                case 'sofort':
```

Comment:
I think you're missing a `default` case. What happens if none matches? At the moment, it falls back to 'PAYPAL' but is it what's intended?